<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyClass</class>
 <widget class="QMainWindow" name="MyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1424</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>机械臂抓取</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>924</width>
      <height>1024</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>922</width>
       <height>1022</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="imagelabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
     border-width: 1px;
     border-style: solid;
     border-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>86</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 255);
font: 75 italic 11pt &quot;Agency FB&quot;;
font: 12pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>物体位姿：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(153, 153, 153);</string>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
   </widget>
   <widget class="QGroupBox" name="groupBoxbutton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>500</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>功能按钮</string>
    </property>
    <widget class="QPushButton" name="beginButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="shotButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>截取</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pointButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>点取</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="operate_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>564</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;AcadEref&quot;;
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>操作指南：</string>
    </property>
   </widget>
   <widget class="QLabel" name="display_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>500</width>
      <height>360</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(255, 85, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;功能按钮说明：&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;开始按钮处理保存的按钮&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;截取按钮处理已展示图片的一部分进行处理&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;点取按钮点取图片中的四个点进行处理&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;菜单按钮说明：&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;file能够打开指定的图片处理或者关闭窗口&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;save能够保存处理后的结果和图片&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;help操作文档手册&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <zorder>groupBoxbutton</zorder>
   <zorder>scrollArea</zorder>
   <zorder>result</zorder>
   <zorder>tableView</zorder>
   <zorder>operate_label</zorder>
   <zorder>display_label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1424</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionOperate_document"/>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>保存</string>
    </property>
    <addaction name="actionPicture"/>
    <addaction name="actiontext"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSave"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionOperate_document">
   <property name="text">
    <string>operate document</string>
   </property>
  </action>
  <action name="actionPicture">
   <property name="text">
    <string>图片</string>
   </property>
  </action>
  <action name="actiontext">
   <property name="text">
    <string>结果</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myclass.qrc"/>
 </resources>
 <connections/>
</ui>
